lib/oli_web/components/delivery/scored_activities/scored_activities.ex

```diff
@@ -28,24 +28,6 @@ defmodule OliWeb.Components.Delivery.ScoredActivities do
 
   alias Oli.Resources.Revision
 
-  prop(assessments, :list, required: true)
-  prop(students, :list, required: true)
-  prop(params, :map, required: true)
-  prop(section, :map, required: true)
-  prop(scripts, :map, required: true)
-  prop(activity_types_map, :map, required: true)
-  prop(view, :string, required: true)
-  prop(ctx, :map, required: true)
-
-  data(table_model, :map)
-  data(total_count, :integer)
-  data(student_emails_without_attempts, :list)
-  data(students_with_attempts_count, :integer)
-  data(total_attempts_count, :integer)
-  data(current_assessment, :map)
-  data(activities, :list)
-  data(preview_rendered, :map)
-
```

Comment:
NIT: Could we use `attr` and not remove this props? 😅

Reply:
Yeah, that would be great. Nowadays the [attr](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#attr/3) is supported for function components, but not yet for live components. There is [an open issue](https://github.com/phoenixframework/phoenix_live_view/issues/2407) that suggests that implementation.
A workaround could be wrapping the live_component with a function component with attrs, but [it is not recommended](https://github.com/phoenixframework/phoenix_live_dashboard/pull/394#issuecomment-1399553639) to do so

Reply:
Oh, I didn't know that! Thanks for the info 😁